plugins/rest/rest.go

```diff
@@ -170,6 +172,17 @@ func New(config []byte, keys map[string]*keys.Config, opts ...func(*Client)) (Cl
 
 	parsedConfig.keys = keys
 
+	// HTTPAuthPlugin.Prepare uses the Bearer token as it is defined in the Config
+	// but the OCIDownloader encoded it to base64 before adding it to the request.
+	// If we want to preserve the old behavior, we need to pre-encode it.
+	if strings.ToLower(parsedConfig.Type) == "oci" &&
+		parsedConfig.Credentials.Bearer != nil &&
+		parsedConfig.Credentials.Bearer.Token != "" {
+
+		token := parsedConfig.Credentials.Bearer.Token
+		parsedConfig.Credentials.Bearer.Token = base64.StdEncoding.EncodeToString([]byte(token))
```

Comment:
What if the path to the token was provided instead of the token itself? This will not work in that case, correct 🤔  This seems to an issue with the current implementation as well.

Reply:
Currently, the OCIDownloader doesn't support `TokenPath` at all. So I was thinking that we wouldn't break any existing behavior. But I agree that it would be inconsistent.

Reply:
I've added logic to encode tokens from the `TokenPath` too 👍